src/pkg/cli/compose/context.go

```diff
@@ -166,6 +167,59 @@ func tryReadIgnoreFile(cwd, ignorefile string) io.ReadCloser {
 	return reader
 }
 
+// writeDockerIgnoreFile writes a default
+// .dockerignore file to the specified directory.
+func writeDockerIgnoreFile(cwd string) error {
+	path := filepath.Join(cwd, ".dockerignore")
+	term.Debug("Writing .dockerignore file to", path)
+	file, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+
+	defer file.Close()
+
+	_, err = file.WriteString(defaultDockerIgnore)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Consider replacing this with `os.WriteFile`. Something like this:
```suggestion
	return os.WriteFile(path, defaultDockerIgnore, 0666)
```

Reply:
Thanks, did know this function existed.